core/src/main/java/com/gcinterceptor/core/GarbageCollectorControlInterceptor.java

```diff
@@ -37,6 +40,12 @@ public GarbageCollectorControlInterceptor(Heap monitor, Executor executor) {
 		this.finished = new AtomicLong();
 		this.shedRequests = new AtomicLong();
 		this.sheddingThreshold = new SheddingThreshold();
+		
+		if (DEBUG_GCI) {
+			csv_file_name = System.getenv("csv_file_name");
```

Comment:
Environment variables capitalized, please.  Also, CSV is too generic. Lets make it more strict.

SHED_RATIO_CSV_FILE

That said, you don´t need the DEBUG_GCI flag anymore. If the SHED_RATIO_CSV_FILE is set, you write this out.

Reply:
Done